### PR TITLE
Fix default HTTPS certificate

### PR DESCRIPTION
**Pull request by Amygos**
If the node's FQDN is used in the default selfsigned certificate, when a valid certificate for the node's FQDN is requested using the `set-certificate` action, `traefik` will not request a new certificate via Let's Encrypt, but will use the already present selfsigned certificate.
This is caused by the fact that in the system is already present a certificate for the requested domain (in the selfsigned certificate) and to avoid this problem the tld `.test` is used in the selfsigned certificate.
The `CN` of the selfsigned certificate will be `FQDN + .test`

### CODE REVIEW

**Review by gsanchietti — COMMENTED** [review]
It looks good to me.
Still, I remember that old Firefox versions did not permit to bypass the control of such certificate.
I need to test it a little bit.

**Review by gsanchietti — APPROVED** [review]
Tests are ok, both on Windows and Linux

File: traefik/imageroot/actions/create-module/20certificate [hunk 1]
```diff
@@ -31,7 +31,7 @@ set -e
 exec 1>&2
 
 NAME="selfsigned"
-FQDN=$(hostname -f)
+FQDN="ns8.local"
```
Comment (DavidePrincipi) [inline traefik/imageroot/actions/create-module/20certificate:34]: The .local suffix is reserved for other purposes. What about appending .test to the FQDN?

```suggestion
FQDN="$(hostname -f).test"
```

Comment (gsanchietti) [reply traefik/imageroot/actions/create-module/20certificate:34]: The browser will complain, but as long as HSTS is not enable on that domain, it should not be a real problem.

Comment (DavidePrincipi) [reply traefik/imageroot/actions/create-module/20certificate:34]: :raised_hands:

**Review by DavidePrincipi — APPROVED** [review]

File: traefik/imageroot/actions/create-module/20certificate (hunk 1 above)
Comment (Amygos) [reply traefik/imageroot/actions/create-module/20certificate]: In the end, I pick the solution proposed by @DavidePrincipi :)

**Pull request merged**